src/components/AvatarWithIndicator.js

```diff
@@ -14,37 +19,116 @@ const propTypes = {
     // avatar size
     size: PropTypes.string,
 
+    // When true, shows sync indicator
```

Comment:
Also picky but do you think it would be more clear to change this to:
```suggestion
    // Whether we show the sync indicator
```